src/dpdb/dpdb.js

```diff
@@ -19,7 +19,8 @@ var DPDB_CACHE = require('./dpdb-cache.js');
 var Util = require('../util.js');
 
 // Online DPDB URL.
-var ONLINE_DPDB_URL = 'https://storage.googleapis.com/cardboard-dpdb/dpdb.json';
+var ONLINE_DPDB_URL =
+  'https://thoragio.github.io/webvr-polyfill-dpdb/dpdb.json';
```

Comment:
OK. All done. GH Pages link is working: https://webvrrocks.github.io/webvr-polyfill-dpdb/dpdb.json

I'll update my PR.